core/jazz_t-vault/swagger/swagger.json

```diff
@@ -0,0 +1,3589 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "version": "2017-01-30T23:30:18Z",
+        "title": "{api_deployment_node_title}",
+        "description": "API specification for jazz t-valut api"
```

Comment:
API specification for jazz t-valut api => Specification for APIs to manage secrets using Jazz & T-Vault

Reply:
Done